lib/flatbuffers/flat_buffers.dart

```diff
@@ -0,0 +1,1317 @@
+// Copyright (c) 2016, the Dart project authors.  Please see the AUTHORS file
```

Comment:
As said, it would be nice to see the actual changes to the original flat_buffers.dart (also possibly makes rebasing these changes onto newer upstream flatbuffers versions easier).

Reply:
Flatbuffers PR for Allocator interface: https://github.com/google/flatbuffers/pull/6397

Reply:
yep, that's exactly what's changed. The final version of the code here is an exact copy of the code in the PR